android/modules/utils/src/java/ti/modules/titanium/utils/UtilsModule.java

```diff
@@ -66,7 +66,7 @@ public TiBlob base64encode(Object obj)
 		String data = convertToString(obj);
 		if (data != null) {
 			try {
-				return TiBlob.blobFromString(new String(Base64.encodeBase64(data.getBytes("UTF-8")), "UTF-8"));
+				return TiBlob.blobFromString(new String(Base64.encode(data.getBytes("UTF-8"), Base64.DEFAULT), "UTF-8"));
```

Comment:
I did a check on this with the following code
```java
				//Current
				Log.d("LogCheck1", new String(Base64.encode(data.getBytes("UTF-8"), Base64.DEFAULT), "UTF-8"));
				//Old and Correct
				Log.d("LogCheck2", new String(org.apache.commons.codec.binary.Base64.encodeBase64(data.getBytes("UTF-8")), "UTF-8"));
				//New and Correct
				Log.d("LogCheck3", new String(Base64.encode(data.getBytes("UTF-8"), Base64.NO_WRAP), "UTF-8"));
```

Using Base64.DEFAULT produces a slightly different String that include line terminators.

I tested it with this code:
```javascript

var encoded = Ti.Utils.base64encode("Hi there Is There\n /n \n THis is \n /n long long \n   long /n helooooooooo Hi "); 

```

Stackoverflow link that is relevant: http://stackoverflow.com/questions/17912119/is-there-any-difference-between-apaches-base64-encodebase64-and-androids-base6